cogs/whisper.py

```diff
@@ -3,7 +3,9 @@
 from discord.ext import commands
 
 class Whisper(commands.Cog):
-    """Whispers, for DCU Bot."""
+    """Whispers, for DCU Bot.
```

Comment:
nit picking, ideally, this change would be a seperate commit, no change required this time however.